### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by wfz666** [XPixelGroup/HAT#162]
Hi, I am now using 4 A100 to train the model, but it is really slow(5 days) for batch size 32, so I would change the batch size to bigger and also the learning rate and milestone. So can you give me suggestions? Thanks!

**Comment by seanko29** [comment]
hi, what configuration did you try with?
I also have the same problem.
